test/spec/desktop/viewportSize.js

```diff
@@ -21,4 +21,14 @@ describe('setViewportSize/getViewportSize', function() {
         });
     });
 
+    it('should set window size exactly when parameter \'type\' is false', function() {
+        return this.client.setViewportSize({
+            width: 500,
+            height: 500
+        }, false).windowHandleSize().then(function(res) {
```

Comment:
we already have the test above, type should be `true` and we should make sure that the window size was changed not the viewport size